### PR TITLE
[examples] Update pusher example to use deploy button

### PR DESCRIPTION
**Pull request by styfle**
Update pusher example to use deploy button, no more secrets

### CODE REVIEW

File: examples/vanilla-pusher-functions/README.md [hunk 1]
```diff
@@ -1,19 +1,9 @@
...
-```
-
-### Deploying From Your Terminal
-
-You can deploy your new project with a single command from your terminal using [Vercel CLI](https://vercel.com/download):
-
-```shell
-$ vercel
-```
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/git/external?repository-url=https%3A%2F%2Fgithub.com%2Fvercel%2Fvercel%2Fedit%2Fmain%2Fexamples%2Fvanilla-pusher-functions&env=APP_ID,KEY,SECRET,CLUSTER&envDescription=Pusher%20Channels&envLink=https%3A%2F%2Fpusher.com%2Fchannels)
```
Comment (lucleray) [inline examples/vanilla-pusher-functions/README.md]: This deploy button doesn't work well 🤔

Comment (lucleray) [reply examples/vanilla-pusher-functions/README.md]:
```suggestion
[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/git/external?repository-url=https%3A%2F%2Fgithub.com%2Fvercel%2Fvercel%2Ftree%2Fmain%2Fexamples%2Fvanilla-pusher-functions&env=APP_ID%2CKEY%2CSECRET%2CCLUSTER&envDescription=Pusher+Channels&envLink=https%3A%2F%2Fpusher.com%2Fchannels)
```

Comment (styfle) [reply examples/vanilla-pusher-functions/README.md]: @lucleray This is generated from https://vercel.com/docs/more/deploy-button

Comment (styfle) [reply examples/vanilla-pusher-functions/README.md]: Oh I see the problem, I used the edit button. Let me update

**Comment by lucleray** [comment]
The `vercel.json` file in the example also needs to updated.

File: examples/vanilla-pusher-functions/README.md (hunk 1 above)
Comment (styfle) [reply examples/vanilla-pusher-functions/README.md]: @lucleray Take a look again 

image

**Comment by styfle** [comment]
@lucleray I removed `vercel.json`, thanks!

**Review by lucleray — APPROVED** [review]

**Review by AndyBitz — APPROVED** [review]

**Pull request merged**